index.html

```diff
@@ -0,0 +1,77 @@
+<body>
```

Comment:
- I kindly recommend that you used HTML 5 semantic header at the start of your HTML file.
- Please add the <link> tag so you can link your style sheet.